packages/slider/src/presenters/stylesheet.js

```diff
@@ -0,0 +1,196 @@
+function stylesheet(props, trackValueRatio, themeData) {
+  const { disabled, hasFocus, hasHover, isPressed } = props;
+
+  const thumbPseudoElements = {
+    microsoft: "::-ms-thumb",
+    mozilla: "::-moz-range-thumb",
+    webkit: "::-webkit-slider-thumb"
+  };
+
+  const trackPseudoElements = {
+    microsoft: "::-ms-track",
+    mozilla: "::-moz-range-track",
+    webkit: "::-webkit-slider-runnable-track"
+  };
+
+  const trackProgressPseudoElements = {
+    // WebKit does not have a pseudo element to target this styling, so we achieve it with fancy styling on track
+    // ::-webkit-slider-runnable-track
+    microsoft: "::-ms-fill-lower",
+    mozilla: "::-moz-range-progress"
+  };
+
+  function browserSpecificPseudoElementRules(
+    browserPseudoElements,
+    rules,
+    singleBrowserInclusions
+  ) {
+    const styles = {};
+
+    Object.entries(browserPseudoElements).forEach(browserPseudoElement => {
+      let pseudoElementRules = rules;
+
+      const browser = browserPseudoElement[0];
+      const pseudoElement = browserPseudoElement[1];
+
+      if (singleBrowserInclusions && singleBrowserInclusions[browser]) {
+        pseudoElementRules = {
+          ...rules,
+          ...singleBrowserInclusions[browser]
+        };
+      }
+
+      styles[`&${pseudoElement}`] = pseudoElementRules;
+    });
+
+    return styles;
+  }
+
+  const baseRules = {
+    WebkitAppearance: "none" /* Hides platform-native styling */,
+    width: "100%" /* Specific width is required for Firefox. */,
+    backgroundColor: "transparent" /* Otherwise white in Chrome */,
+    boxSizing: "content-box",
+    height: "20px",
+    margin: 0,
+    position: "relative",
+    outline: "none",
+
+    "&::-ms-tooltip": {
+      display: "none"
+    },
+
+    "&::-moz-focus-outer": {
+      border: 0
+    }
+  };
+
+  const thumbDisabledRules = disabled
+    ? {
+        opacity: themeData["slider.disabled.opacity"]
```

Comment:
for the `disabled` role there is a universal one `"component.disabled.opacity"` as I think all the inputs and controls use the same `disabled` values.